translations/monero-core_pl.ts

```diff
@@ -994,158 +994,158 @@
     <message>
         <location filename="../pages/Settings.qml" line="96"/>
         <source>Create view only wallet</source>
-        <translation type="unfinished"></translation>
+        <translation>Utwórz portfel tylko do podglądu</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="310"/>
         <location filename="../pages/Settings.qml" line="340"/>
         <source>(optional)</source>
-        <translation type="unfinished">(opcjonalnie)</translation>
+        <translation>(opcjonalne)</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="145"/>
         <source>Rescan wallet balance</source>
-        <translation type="unfinished"></translation>
+        <translation>Zaktualizuj saldo portfela</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="154"/>
         <source>Error: </source>
-        <translation type="unfinished"></translation>
+        <translation>Błąd: </translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="159"/>
         <source>Information</source>
-        <translation type="unfinished">Informacja</translation>
+        <translation>Informacja</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="303"/>
         <source>Blockchain location</source>
-        <translation type="unfinished"></translation>
+        <translation>Lokalizacja blockchaina</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="363"/>
         <source>Username</source>
-        <translation type="unfinished"></translation>
+        <translation>Nazwa użytkownika</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="372"/>
         <source>Password</source>
-        <translation type="unfinished">Hasło</translation>
+        <translation>Hasło</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="397"/>
         <source>Connect</source>
-        <translation type="unfinished"></translation>
+        <translation>Połącz</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="421"/>
         <source>Layout settings</source>
-        <translation type="unfinished"></translation>
+        <translation>Ustawienia wyglądu</translation>
     </message>
     <message>
         <location filename="../pages/Settings.qml" line="439"/>
         <source>Custom decorations</source>
-        <translation type="unfinished"></translation>
+        <translation>Wbudowana skórka</translation>
```

Comment:
Shouldn't it be "Zmień skórkę"?

Reply:
Well, it could be, but I asked on #monero-gui how "Custom decorations" work and it appears that it's not configurable in any way. The main reason for that feature is that on some systems you have minimize, maximize and closing buttons in the upper left corner and on others they are in upper right corner and this option unifies that. Additionally it makes the window black. So I wanted to emphasize that it's something that's built in and is not configurable. With "Zmień skórkę" (which is translated directly to: change theme) I'd expect a popup that would allow me to configure that theme (because: "change" to what?).

But of course I'm not hard on that tranlation so if you disagree, I can change it and users shouldn't have problems with your translation either.

Reply:
OK. I think that original phrase would be updated.